tests/contrib/scrapy/test_pipeline_cerberus.py

```diff
@@ -135,4 +143,58 @@ def test_validation_from_url(mocker):
     pipe.process_item(test_item, None)
 
     assert "spidermon/validation/items/errors" in pipe.stats.stats.get_stats()
-    assert pipe.stats.stats.get_stats().get("spidermon/validation/validators/testitem/cerberus", True)
+    assert pipe.stats.stats.get_stats().get(
+        "spidermon/validation/validators/testitem/cerberus", True
+    )
+
+
+@pytest.mark.skipif(
+    sys.version_info < (3, 4), reason="mock requires python3.4 or higher"
+)
+def test_pipelines_with_cerberus_mocked_true_case(mocker):
+    mock_cerberus = mock.patch("spidermon.contrib.scrapy.pipelines.CerberusValidator")
+
+    class Cerberus(object):
+        def validate(self):
+            return (False, {})
+
+    mock_cerberus.return_value = Cerberus()
```

Comment:
`mock_cerberus` is already a mock, so
```
mock_cerberus.validate.return_value = (False, {})
```
should be fine.

Besides that, when patching, try to use `autospec`
https://docs.python.org/3/library/unittest.mock.html#patch